modules/sql-engine/src/integrationTest/sql/group1/types/time/time_parsing.test

```diff
@@ -28,84 +28,88 @@ SELECT '14:42:04.999000'::TIME(6)::VARCHAR
 ----
 14:42:04.999
 
-query T
+statement error: Invalid TIME value, '1'
 SELECT '1'::TIME
-----
-01:00:00
 
-query T
+statement error: Invalid TIME value, '11'
 SELECT '11'::TIME
-----
-11:00:00
 
-statement error
+statement error: Invalid TIME value, '11:'
 SELECT '11:'::TIME
 
-query T
+statement error: Invalid TIME value, '11:1'
 SELECT '11:1'::TIME
-----
-11:01:00
 
-statement error
+statement error: Invalid TIME value, '11:01'
+SELECT '11:01'::TIME
+
+statement error: Value of HOUR field is out of range in '25:42:04.500'
+SELECT '25:42:04.500'::TIME(3)
+
+statement error: Invalid TIME value
 SELECT '23:60:00'::TIME
-----
 
-# https://issues.apache.org/jira/browse/IGNITE-25000 Negative time components / out of range values
-skipif ignite3
-statement error
+statement error: Invalid TIME value, '23:00:60'
 SELECT '23:00:60'::TIME
 
-# https://issues.apache.org/jira/browse/IGNITE-25000 Negative time components / out of range values
-skipif ignite3
-statement error
+statement error: Invalid TIME value, '22:60:60'
 SELECT '22:60:60'::TIME
 
-statement error
+statement error: Invalid TIME value, '23:59:60'
 SELECT '23:59:60'::TIME
-----
 
-# https://issues.apache.org/jira/browse/IGNITE-25000 Negative time components / out of range values
-skipif ignite3
-statement error
+statement error: Invalid TIME value, '100:42:04.500'
+SELECT '100:42:04.500'::TIME(3)
```

Comment:
It's the same type of error as on line 47, however, the message has different format